api/src/main/java/com/epam/pipeline/manager/pipeline/PipelineRunManager.java

```diff
@@ -800,10 +802,15 @@ private boolean checkRunForSensitivity(final Map<String, PipeConfValueVO> parame
         List<Long> datastorageIds = MapUtils.emptyIfNull(parameters).entrySet().stream()
                 .filter(v -> v.getKey().equals(CP_CAP_LIMIT_MOUNTS))
                 .map(Map.Entry::getValue)
-                .flatMap(pipeConfValueVO -> Arrays.stream(
-                        StringUtils.commaDelimitedListToStringArray(pipeConfValueVO.getValue()))
+                .flatMap(pipeConfValueVO -> {
+                            String limitMounts = pipeConfValueVO.getValue();
+                            if (limitMounts.equalsIgnoreCase(LIMIT_MOUNTS_NONE)) {
```

Comment:
It is better to flip equals to prevent NPE :  `LIMIT_MOUNTS_NONE. equalsIgnoreCase(limitMounts)`